Bottom: fd3530a3b92f2439da47763d71e6dba3b6cd93ec
Top:    28fb1bea93c85f9c7ce7d15fb52d7ad7ba9515da
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-09 13:02:45 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 59a9a29..7843ca4 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -332,13 +332,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
